cmake/IgnUtils.cmake

```diff
@@ -952,12 +952,58 @@ function(ign_create_core_library)
 
   #------------------------------------
   # Create the target for the core library, and configure it to be installed
-  _ign_add_library_or_component(
-    LIB_NAME ${PROJECT_LIBRARY_TARGET_NAME}
-    INCLUDE_DIR "${PROJECT_INCLUDE_DIR}"
-    EXPORT_BASE IGNITION_${IGN_DESIGNATION_UPPER}
-    SOURCES ${sources}
-    ${interface_option})
+
+  # Support "gz-"
+  if(${PROJECT_LIBRARY_TARGET_NAME} MATCHES "^gz-")
+    _ign_add_library_or_component(
+      LIB_NAME ${PROJECT_LIBRARY_TARGET_NAME}
+      INCLUDE_DIR "${PROJECT_INCLUDE_DIR}"
+      EXPORT_BASE GZ_${IGN_DESIGNATION_UPPER}
+      SOURCES ${sources}
+      ${interface_option})
+
+    # For ticktocking: Export an "ignition-" target as well, allowing linking against
```

Comment:
this extra target isn't discussed in the pull request description. Is it necessary for this pull request?

Reply:
You're right, it isn't. Reverted. 64bad566376334eab09bc57f77b1b6c61336e179